corehq/motech/repeaters/tests/test_models.py

```diff
@@ -229,38 +229,45 @@ def set_next_attempt_at(repeater, when):
         repeater.save()
 
 
-class ResponseMock:
+class Unknown:
     pass
 
 
 class IsResponseTests(SimpleTestCase):
 
     def test_has_text(self):
-        resp = ResponseMock()
-        resp.text = '<h1>Hello World</h1>'
+        resp = Unknown()
```

Comment:
Why was this changed from `ResponseMock` (which looks like it was moved below) to `Unknown`?

Reply:
`corehq/motech/repeaters/tests/test_models.py` still defines and uses a `ResponseMock` class, which mocks a Response object. I added the `Unknown` class because it does not mock a Response object.